bevy_lint/src/lints/nursery/duplicate_bevy_dependencies.rs

```diff
@@ -205,21 +205,18 @@ fn minimal_lint(
                 return node.id.repr.split('#').nth(1).map(|version| vec![version]);
             }
             // Extract versions from external crates
-            if let Some((id, _)) = node.id.repr.split_once('@') {
-                if bevy_dependents
+            if let Some((id, _)) = node.id.repr.split_once('@')
+                && bevy_dependents
                     .keys()
                     .any(|crate_name| id.ends_with(crate_name))
```

Comment:
I merged these two `if` statements together into one, but didn't change any of the logic, so it should behave the same.